### PR TITLE
Move authenticate with AD to a folder

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
Move the 'authenticate with AD ' article to a folder so that is appears in the menu. Belongs to: https://github.com/umbraco/UmbracoDocs/issues/3994

### CODE REVIEW

File: Reference/Security/index-vpost-7.5.0.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ versionFrom: 7.5.0
 
 Umbraco 7.5.0+ comes with a built-in `IBackOfficeUserPasswordChecker` for **Active Directory**: `Umbraco.Core.Security.ActiveDirectoryBackOfficeUserPasswordChecker`.
 
-Find out how to activate the [ActiveDirectoryBackOfficeUserPasswordChecker](authenticate-with-AD.md)
+Find out how to activate the [ActiveDirectoryBackOfficeUserPasswordChecker](Authenticate-with-Active-Directory/authenticate-with-AD.md)
```
Comment (eshanrnh) [inline Reference/Security/index-vpost-7.5.0.md:9]:
```suggestion
Find out how to activate the [ActiveDirectoryBackOfficeUserPasswordChecker](Authenticate-with-Active-Directory/Authenticate-With-AD-vpre8.md)
```

**Review by eshanrnh — COMMENTED** [review]
The PR looks good, @erikjanwestendorp I have added a suggestion to reference the v7.0.0 to v7.5.0 article instead of v8 article 🙂

Also, can you update the link in index-v7.md to include the folder in the path so that we don't have a broken link?

**Comment by erikjanwestendorp** [comment]
@eshanrnh Thanks for your review, just updated the PR with your suggestion and also updated the link in index-v7.md.

**Review by eshanrnh — APPROVED** [review]
Thanks @erikjanwestendorp 🙌 Merging it in.

**Pull request merged**